app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -210,6 +210,11 @@ function editItem(item: DisplayItem) {
 	const pkField = relationInfo.value.relatedPrimaryKeyField.field;
 
 	newItem = false;
+	for (const entry in item) {
+		if (entry !== pkField) {
+			delete item[entry];
+		}
+	}
```

Comment:
Instead of removing every non-primary key field from the `item` object, couldn't we just set `editsAtStart.value` to a new object with just the pkField? 🤔 

```js
editsAtStart.value = { [pkField]: item[pkField ]}
```

Reply:
Thanks for the comments, simplified the implementation with this suggestion as we just want to pass the key of the relation collection item.